UE/Application/Sms.cpp

```diff
@@ -8,6 +8,7 @@ Sms::Sms(common::PhoneNumber from, std::string text)
     : from(from),
       text(text)
 {
+    this->read = false;
```

Comment:
`this` is not necessary here.
```suggestion
    read = false;
```